ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -99,10 +99,9 @@ ghcLibParserHsSrcDirs forParserDepends ghcFlavor lib =
         , "compiler/llvmGen"
         , "compiler/rename"
         , "compiler/stgSyn"
-        , "compiler/stranal"
-        , "compiler/nativeGen"
-        ] ++
-        [ "compiler/deSugar" | ghcFlavor `elem` [GhcMaster, Ghc8101] && not forParserDepends]
+        , "compiler/stranal" ] ++
+        [ "compiler/nativeGen" | ghcFlavor /= GhcMaster] ++ -- Since 1/4/2020. See https://gitlab.haskell.org/ghc/ghc/commit/d561c8f6244f8280a2483e8753c38e39d34c1f01.
```

Comment:
```suggestion
        [ "compiler/nativeGen" | ghcFlavor /= GhcMaster] ++ -- Since 2020-01-04. See https://gitlab.haskell.org/ghc/ghc/commit/d561c8f6244f8280a2483e8753c38e39d34c1f01.
```

Reply:
Ok